plugins/plugin-vue/plugin.js

```diff
@@ -51,18 +51,20 @@ module.exports = function plugin(snowpackConfig) {
       }
 
       const output = {
-        '.js': {code: '', map: ''},
-        '.css': {code: '', map: ''},
+        '.js': {contents: '', code: '', map: ''},
```

Comment:
re larger comment above: a plugin should only ever return `code` or `content`, not both.